test/e2e_node/e2e_node_suite_test.go

```diff
@@ -84,7 +84,7 @@ func registerNodeFlags(flags *flag.FlagSet) {
 	flags.StringVar(&framework.TestContext.SystemSpecName, "system-spec-name", "", "The name of the system spec (e.g., gke) that's used in the node e2e test. The system specs are in test/e2e_node/system/specs/. This is used by the test framework to determine which tests to run for validating the system requirements.")
 	flags.Var(cliflag.NewMapStringString(&framework.TestContext.ExtraEnvs), "extra-envs", "The extra environment variables needed for node e2e tests. Format: a list of key=value pairs, e.g., env1=val1,env2=val2")
 	flags.StringVar(&framework.TestContext.SriovdpConfigMapFile, "sriovdp-configmap-file", "", "The name of the SRIOV device plugin Config Map to load.")
-	flag.StringVar(&framework.TestContext.ClusterDNSDomain, "dns-domain", "cluster.local", "The DNS Domain of the cluster.")
+	flag.StringVar(&framework.TestContext.ClusterDNSDomain, "dns-domain", "", "The DNS Domain of the cluster.")
```

Comment:
Thanks for doing this.
I see the test failures like https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-features/1349429706050506752 as an example.
One of failed tests sets `hostFQDN` like https://github.com/kubernetes/kubernetes/blob/5c58b222516c0c8547d2f699b7582db9df09df8d/test/e2e_node/pod_hostnamefqdn_test.go#L117
If `framework.TestContext.ClusterDNSDomain` becomes empty like this PR, the hostFQDN becomes like `foo-pod.foo-subdomain.foo-ns.svc.` without `cluster.local`.
The FQDN is different from common cases I feel like https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/
Can we enable`cluster.local` instead?

/cc @oomichi

Reply:
@oomichi we could, although I am hesitant to require that for these tests to run successfully as that is not the default for the kubelet. If we were to set to `cluster.local` we would do so here: https://github.com/kubernetes/test-infra/blob/c094f147c10f9b667472e81a5b36e12a61446e37/config/jobs/kubernetes/sig-node/node-kubelet.yaml#L127

Reply:
I see, thanks for your explanation.
I guess it is ideal to change the default value to "cluster.local" on kubelet side at the end of day, but current change is fine for me to solve test failure at this time.

/triage accepted
/lgtm